site/content/posts/a-seal-of-approval.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "A Seal of Approval: Project Pinniped"
+slug: a-seal-of-approval
+date: 2020-11-12
+author: Pablo Schuhmacher
+image: /img/logo.svg
+excerpt: "Pinniped intends to bring that dream state — log in once and you’re done — to reality."
+tags: ['Pablo Schuhmacher']
+---
+
+Kubernetes, containers, microservices: They’ve all turned conventional application development wisdom inside out. But for all the wonders introduced and new technologies released, there are still a few things that remain difficult, cumbersome, or just really really frustrating when it comes to Kubernetes. We have set out to make one of those things easier and more understandable: authentication.
+
+In a perfect world, you would be able to use a single authentication process of your choice to log in to all of your Kubernetes clusters, including on-premises and managed cloud environments. This process would be highly secure, easy to configure, and tightly integrated with standard upstream identity providers. The reality is quite different. Authentication can be a tricky affair.
+
+[Pinniped](https://pinniped.dev/), a newly released VMware-originated open source project, intends to bring that dream state — log in once and you’re done — to reality.
+
+### The state of Kubernetes
+
+Kubernetes offers a wide range of authentication backends, but the end-to-end login flow for your clusters is up to you. Kubernetes itself handles only credential validation, and usually requires extra tools and configuration to integrate with external identity providers. Unfortunately, this means that in practice many clusters wind up with less secure options, like shared “admin” certificates.
+
+Even if you are consuming a managed Kubernetes solution or distribution that provides integrated authentication, the authentication configuration is often controlled solely by the provider. As a consumer of Kubernetes in these situations, there hasn’t been a single, unified way to customize authentication. In some cases, users need to know how to log in several different ways to access multiple clusters.
+
+Pinniped delivers a consistent user authentication experience in Kubernetes that prioritizes security, interoperability, and low-effort management at scale. Using Pinniped, you’re able to:
+
+- Install and integrate with nearly any cluster in one step
```

Comment:
This bulleted list is not displaying quite right:

![image](https://user-images.githubusercontent.com/3210334/98876364-fd34e900-2432-11eb-9a72-b28ee2b598c9.png)

Reply:
Oh, looks like this is a known issue (I can read my email after all).